src/_utils/illustration.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import isEmpty from 'lodash.isempty'
+import styled from 'styled-components'
+
+export type IllustrationProps = Readonly<{
+  title?: string
+}>
+
+type BaseIllustrationProps = IllustrationProps &
+  Readonly<{
+    children: React.ReactNode
+  }>
+
+const StyledIllustration = styled.svg`
+  max-height: 33vh;
+  width: 100%;
+  object-fit: contain;
+`
+
+export const Illustration = ({ children, title = '' }: BaseIllustrationProps): JSX.Element => (
+  <StyledIllustration
+    viewBox="0 0 375 204"
```

Comment:
would declare as default props

Reply:
the viewBox is here to have the basic aspect ratio. If we have a width under 375 it'll break using min-width. We should not override these imho